### PR TITLE
Additional DevOps Documentation

### PR DESCRIPTION
**Pull request by DanPaseltiner**
This PR adds the following to devops.md

- [x] table of contents
- [x] environments section
- [x] deployments sections
- [x] environment variables section
- [x] condenses content related to upgrading the PHDI/DIBBs version

### CODE REVIEW

File: docs/devops.md [hunk 1]
```diff
@@ -30,12 +45,60 @@ Our deployment pipeline is defined in the YAML file `phdi-azure/.github/workflow
...
+
+The deployment pipeline is capable of deploying the starter kit to any number of environments. This allows users to flexibly configure however many distinct instances of the starter kit they need (dev, test, staging, prod, etc.). Currently all environments are deployed within a single Azure resource group. The environment name is included in the names of individual resources to distinguish which environment they belong to. If the starter kit is initially deployed by following the [Implementation Guide](implementation-guide.md), as recommended, it will only have a `dev` environment. To create additional environments follow the steps below:
+1. Navigate to `Settings` on your version of the `phdi-azure` repository in GitHub.
+2. Click on `Environments` in the side bar.
+3. Click on `New Environment` in the top right.
+![make-new-env-1](./images/make-new-env-1.png)
+4. Enter the name of your new environment and click `Configure environment`.
+![make-new-env-2](./images/make-new-env-2.png)
+
+After following these steps to initialize a new environment see the [Deployments](#deployments) sections below for guidance on how to deploy the starter kit to it.
```
Comment (bamader) [inline docs/devops.md:59]: I think this is a really good addition for this document

**Review by bamader — APPROVED** [review]

File: docs/devops.md (hunk 1 above)
Comment (DanPaseltiner) [reply docs/devops.md:59]: Thanks!

**Review by BradySkylight — APPROVED** [review]
This is pretty awesome.  I love the screen shots that make it very clear.  Nice work Dan!

**Review by gordonfarrell — APPROVED** [review]
Looks great, Dan! Really good detail in your instructions and great background/whys as well

**Pull request merged**